pyvista/core/pointset.py

```diff
@@ -171,14 +171,13 @@ def translate(
 
         Parameters
         ----------
-        xyz : list or tuple or np.ndarray
-            Length 3 list, tuple or array.
+        xyz : list or tuple or numpy.ndarray
+            Cartesian values to displace. Length 3 list, tuple or array.
```

Comment:
Forgot this one, sorry.
```suggestion
            Cartesian values to displace with. Length 3 list, tuple or array.
```